content/blog/2024-06-12-rustl-2024-unconference.md

```diff
@@ -0,0 +1,200 @@
++++
+title = "Report on the RustNL 2024 Unconference"
+authors = ["Olivier Faure"]
+date = "2024-06-12 18:00:00"
++++
+
+On May 7 and 8, 2024, folks from the Linebender team went to [the RustNL conference](https://2024.rustnl.org/) in Delft.
+We listened to talks, Raph gave one, Rik Arends did the "hot reloading my entire editor live" thing that makes every other Rust GUI developer jealous, it was an all-around good time.
+
+(Videos for individual talks aren't up yet, but there are livestream recordings of [Day 1](https://www.youtube.com/watch?v=XLefuzE-ABU) and [Day 2](https://www.youtube.com/watch?v=521NfGf7AR0) on Youtube.)
+
+After RustNL, though, another less-documented event called [The Unconference](https://2024.rustnl.org/unconf/). It was from roughly the same organizers and took place in the same city, but the format was fairly different.
+
+The Unconference wasn't livestreamed, and I haven't found any public discussion of it, so I thought I'd take the time to describe a bit of what was going.
+
+
+## The format
+
+The "Unconference" was two days of loosely-structured chats between members of various Rust project teams where people talked about ecosystem collaboration, complained to Lang Team members that their favourite feature wasn't implemented yet, [went to pester everyone else about variadic generics](https://poignardazur.github.io/2024/05/25/report-on-rustnl-variadics/), and other good fun.
+
+Attendees were split into three teams:
+
+- The Rust project.
+- GUI and Applications.
+- Embedded development.
+
+Discussions inside of these teams was mostly self-organized, with cross-team discussion mostly taking place during lunch and recesses.
+
+
+### On the tyranny of struturelessness
+
+*Note: This section is very much my personal opinion and doesn't represent the Linebender project as a whole.*
+
+I'm not sure the self-organized format worked well.
+
+From what I saw, the Embedded team took well to it, and the Rust team was kept productive thanks to Alice Cecile's efforts in marshalling everyone.
+
+In the case of the GUI team, people were spread in a very large room, which should have been conducive to small side-discussions and people splitting up to talk about the problems that interested them. Instead, there was an unspoken accord to progress through agenda items one by one, with a few people dominating the discussions on these items.
+
+Because those people we spread around a large room, they had to talk loudly to address each other, which left little room for side-discussions.
```

Comment:
```suggestion
Because those people were spread around a large room, they had to talk loudly to address each other, which left little room for side-discussions.
```